tableau-specialist/1.0/modules/ROOT/pages/tableau-specialist-connector-examples.adoc

```diff
@@ -0,0 +1,957 @@
+http://www.mulesoft.org/schema/mule/tableau http://www.mulesoft.org/schema/mule/tableau/current/mule-tableau.xsd">
+	<configuration-properties file="project-properties.properties"/>
+	<tableau-specialist:tableau-specialist-config name="Tableau_Specialist_Configuration" doc:name="Tableau Specialist Configuration" doc:id="6c0dc7f6-facd-4f38-acbe-b598686b2fb6" >
+		<tableau-specialist:basic-username-password-connection serverUrl="${credentials.serverUrl}" apiVersion="${credentials.apiVersion}" contentUrl="${credentials.contentUrl}" username="${credentials.username}" password="${credentials.password}" />
+	</tableau-specialist:tableau-specialist-config>
+	<http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config" doc:id="6dd0ac5f-0a8b-421f-bd81-ceb4f450a792" >
+		<http:listener-connection host="0.0.0.0" port="8081" />
+	</http:listener-config>
+	<flow name="tableau-demo-publish-workbook" doc:id="960ecd4c-91fd-4884-bb94-d2aedbcf94ec" >
+		<http:listener doc:name="Listener" doc:id="75a721e3-77fc-4c6c-93b4-6bc5b2e0a1cb" config-ref="HTTP_Listener_config" path="/publish-workbook"/>
+		<file:read doc:name="Read" doc:id="e5342f6f-02a1-4c38-982a-b780a7dcaaac" config-ref="File_Config" path="test-workbook.twbx"/>
+		<ee:transform doc:name="Transform Message">
+                <ee:variables>
+                    <ee:set-variable variableName="fileContent"><![CDATA[%dw 2.0
+output application/java
+---
+payload]]></ee:set-variable>
+                </ee:variables>
+            </ee:transform>
+        <ee:transform doc:name="Transform Message">
+				<ee:message>
+					<ee:set-payload><![CDATA[%dw 2.0
+output application/json
+---
+{
+	workbook: {
+		name: "Publish_workbook_test"
+	}
+}]]></ee:set-payload>
+				</ee:message>
+			</ee:transform>
+			<tableau-specialist:publish-workbook doc:name="Publish workbook" doc:id="74fe54f8-8165-4183-b7fd-1a1ed63a9428" config-ref="Tableau_Specialist_Configuration" overwriteFlag="true">
+			<tableau-specialist:workbook-request-options >
+					<tableau-specialist:publish-workbook-request-body workbookFileContent="#[vars.fileContent]" workbookFileName="test-workbook.twbx" />
+			</tableau-specialist:workbook-request-options>
+		</tableau-specialist:publish-workbook>
+		<ee:transform doc:name="Transform Message" doc:id="bc5a4103-da41-48ac-b08a-8676684d3965" >
+			<ee:message >
+				<ee:set-payload ><![CDATA[%dw 2.0
+output application/json
+---
+payload]]></ee:set-payload>
+			</ee:message>
+		</ee:transform>
+	</flow>
+</mule>
+----
+
+=== Steps for Running This Example
+
+. Verify that your connector is configured.
+. Save the project.
+. From a web browser, test the application by entering `http://localhost:8081/publish-workbook`.
+
+
+[[query-projects]]
+== Query Projects - Rest Configuration
+
+This Mule flow returns a list of projects onto the specified site, with optional
+parameters for specifying the paging of large results.
```

Comment:
Suggestion: This Mule flow returns a list of projects from the specified site. It uses optional parameters to specify paging for large results.

(Not sure what onto the specified site means - my suggestion on this might be wrong.)